frontend/js/libs/mediaelement/mediaelement-and-player.4.2.6.js

```diff
@@ -0,0 +1,8522 @@
+/*!
```

Comment:
Why do you add this?

Reply:
That was added from the MEJS player from the official branch, I put the non minified version to use the source map.

Reply:
Sorry I didn't Understand before!, those are from the official branch MEJS branch, I will delete this two files